cmd/migrate-v2/Makefile

```diff
@@ -0,0 +1,122 @@
+.PHONY: test test-unit test-integration test-verbose test-coverage clean build help
+
+# Build the migrate binary
+build:
+	@echo "Building migrate-v2 binary..."
```

Comment:
You can clean up all the `@echo` stuff by replacing them with bash comments. Much cleaner code, slightly different output.

```diff
- @echo "Building migrate-v2 binary..."
+ # Building migrate-v2 binary...
```